### PR TITLE
96 error message component

### LINKED ISSUE #96: Create ErrorMessage component
This will be pretty similar to #90!

Use a [Semantic UI Message](https://react.semantic-ui.com/collections/message/) to create a beautiful error message that we can use across the app! Have a tutū with some visual stylings and run your ideas past a senior developer for approval.

This will receive a `message` through props that should be displayed.

### PR DESCRIPTION
**Pull request by tamari-gray**
closes #96

### CODE REVIEW

File: client/api/api.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import request from 'superagent'
```
Comment (emilyparkes) [inline client/api/api.js:1]: This file seems unrelated to this ticket.

File: client/api/__mocks__/api.js [hunk 2]
```diff
@@ -1,7 +1,17 @@
 export function addFood () {
```
Comment (emilyparkes) [inline client/api/__mocks__/api.js]: This file seems unrelated to this ticket.

File: tests/client/components/FoodDetails.test.js [hunk 3]
```diff
@@ -5,6 +5,8 @@ import { render, shallow } from 'enzyme'
 import configureStore from 'redux-mock-store'
```
Comment (emilyparkes) [inline tests/client/components/FoodDetails.test.js]: This file seems unrelated to this ticket.

File: client/components/FoodDetails.jsx [hunk 4]
```diff
@@ -1,38 +1,64 @@
 import React from 'react'
```
Comment (emilyparkes) [inline client/components/FoodDetails.jsx]: This file seems unrelated to this ticket.

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: client/components/FoodDetails.jsx (hunk 4 above)
Comment (tamari-gray) [reply client/components/FoodDetails.jsx]: ? i think this line needs to be there

File: tests/client/components/ErrorMessage.test.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { Message } from 'semantic-ui-react'
+
+import ErrorMessage from '../../../client/components/ErrorMessage'
+// import { shallow } from 'enzyme'
```
Comment (emilyparkes) [inline tests/client/components/ErrorMessage.test.js]: remove comment please

**Review by emilyparkes — APPROVED** [review]
